src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/search_sorted.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_op_tests/search_sorted.hpp"
+
+namespace ov {
+namespace test {
+
+INSTANTIATE_TEST_SUITE_P(smoke_SearchSortedTest,
+                         SearchSortedLayerTest,
+                         ::testing::Combine(::testing::ValuesIn(SearchSortedLayerTest::GenerateParams()),
+                                            testing::Values(ElementType::f32, ElementType::f16),
```

Comment:
Please add i8 at least

Reply:
HI, I have added tests for i64 and u32 - since they are used in the model that needs SearchSorted.

I cannot add currently test for i8 due to a bug in data_utils.hpp:317, which causes generate_input to loop infinitively. After fixing it, it will work, but fixing that bug is orthogonal to this PR and should be done in separate PR.